alacritty.yml

```diff
@@ -33,6 +33,9 @@ window:
   # Setting this to false will result in window without borders and title bar.
   decorations: true
 
+  # When true, alacritty starts maximized.
+  start_maximized: false
```

Comment:
Please also add this to `alacritty_macos.yml`.

Reply:
Done

Reply:
Must also be added to `alacritty_windows.yml`